nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -106,6 +106,28 @@ def get_weight(self, node_with_weight: NNCFNode, weight_port_id: int, model: ov.
         weight_tensor = get_const_value(weight_node)
         return Tensor(weight_tensor)
 
+    def get_weight_dtype(
+        self, node_with_weight: NNCFNode, weight_port_id: int, model: ov.Model, graph: NNCFGraph
+    ) -> TensorDataType:
+        weight_name = node_with_weight.layer_attributes.constant_attributes[weight_port_id]["name"]
```

Comment:
I think suboptimal constant type definition may be a common problem for NNCF. What do you think about adding dtype to constant_attributes and reusing it in the algorithm?

Reply:
Added dtype to OV node constant_attributes